src/ui/color_trigger.ui

```diff
@@ -3,98 +3,98 @@
  <class>color_trigger</class>
  <widget class="QDialog" name="color_trigger">
   <property name="windowModality">
-   <enum>Qt::ApplicationModal</enum>
+   <enum>Qt::NonModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>364</width>
-    <height>270</height>
+    <width>757</width>
+    <height>435</height>
    </rect>
   </property>
+  <property name="windowTitle">
+   <string>ANSI 256 Color chooser</string>
+  </property>
+  <property name="windowIcon">
+   <iconset resource="../mudlet.qrc">
+    <normaloff>:/icons/mudlet_color_256.png</normaloff>:/icons/mudlet_color_256.png</iconset>
```

Comment:
It's a good try but it doesn't look that great (not that I can make a better one). Do we really need 256 in the icon?

Reply:
It drops the oxygen colour picker icon on the Mudlet arch icon - the "256" is a hang-over from a long ago previous attempt that switched between a modified version of the original 16 colour one and a new one for the other 240 colours. I'll get rid of the red text as it isn't needed...

Reply:
Resolved in so far as the red 256 has gone from the otherwise serviceable icon - it is a 48x48 one so will want redoing in a larger size but that is a different ball game and need not be an issue here.